### PR TITLE
Module - update mbed-drivers dep

### PR DESCRIPTION
**Pull request by 0xc0170**
@BlackstoneEngineering

### CODE REVIEW

File: module.json [hunk 1]
```diff
@@ -16,9 +16,8 @@
     }
   ],
   "dependencies": {
-    "mbed-drivers": "~0.6.4",
-    "uvisor-lib": "~0.7.14"
+    "mbed-drivers": "~0.10.0"
```
Comment (bogdanm) [inline module.json]: Just thinking outlout here: shouldn't this be just `*` by default ? Unless the API in `mbed-drivers` changes in a breaking way, `*` should work. When/if the API changes in a breaking way, we can correct the dependency.

Comment (0xc0170) [inline module.json]: I had the same thought previously, yet not being brave enough to use * :grinning:  Shall we?

Comment (bogdanm) [inline module.json]: Yes, let's. I think we should encourage `*` in top level modules (examples and apps) as much as possible, in order to minimize semantic versioning hassles for people writing mbed OS apps.

**Comment by 0xc0170** [comment]
Amended, dep is now *

**Comment by bogdanm** [comment]
+1

**Comment by BlackstoneEngineering** [comment]
Guys, i'm not maintaining this, if you need to do PR's in the future just do it.

**Comment by 0xc0170** [comment]
@BlackstoneEngineering you are author in module.json file, although your name there is not correct, I'll send fix for that

**Comment by BlackstoneEngineering** [comment]
@0xc0170 right, made @bogdanm co-owner of this repo, please feel free to update this as necessary, I am no longer watching this repo so in the future, like nike says, just do it.

**Pull request merged**